Sources/Hub/HubApi.swift

```diff
@@ -243,9 +245,8 @@ public extension Hub {
     }
 }
 
-public extension Array<String> {
+public extension [String] {
```

Comment:
This is unrelated, but ok.

Reply:
This has been processed by the swift-format tool.

Reply:
Yeah, I don't use it. I write what feels clearer in each case, I tend to use the short form for variables, and the spelled out format when working with the type itself, such as in extensions. But it's ok.

Reply:
Can we introduce swift-format to unify the code style? Similar to https://github.com/nicklockwood/SwiftFormat?tab=readme-ov-file#git-pre-commit-hook or https://github.com/apple/swift-format with git pre commit hook